### PR TITLE
Fix #includes referencing inttypes.h

### PR DESCRIPTION
**Pull request by Susurrus**
@LorenzMeier here's your PR for #170!

This change has been limited to the 1.0 version of MAVLink, as this code change makes a firm cutoff on supported compilers at VS2010. I don't know if anyone was actually using those compilers with this library, because the old code actually did the opposite of what it should have done. And neither `<inttypes.h>` **nor** `<stdint.h>` are available in these older versions. So this really doesn't break the status quo, it just cleans it up and makes the errors more sane for users trying to compile with older MSVC compilers.

This has been tested with VS2013 as well as Microchip's XC16 compiler.

### CODE REVIEW

File: pymavlink/generator/C/include_v1.0/mavlink_helpers.h [hunk 1]
```diff
@@ -226,7 +226,7 @@ MAVLINK_HELPER void mavlink_update_checksum(mavlink_message_t* msg, uint8_t c)
  * A typical use scenario of this function call is:
  *
  * @code
- * #include <inttypes.h> // For fixed-width uint8_t type
+ * #include <mavlink.h>
```
Comment (LorenzMeier) [inline pymavlink/generator/C/include_v1.0/mavlink_helpers.h]: Why this change? I'm slightly worried about recursive includes there - I recall some platforms having some order dependencies.

Comment (Susurrus) [inline pymavlink/generator/C/include_v1.0/mavlink_helpers.h]: Note that this is just example code. Would it be better if I just removed the line? It's not explicitly required for the code that follows.

Now the reason I replaced it with a mavlink include is because that will be required for someone to compile the example (but it won't compile for other reasons as well).

Comment (LorenzMeier) [inline pymavlink/generator/C/include_v1.0/mavlink_helpers.h]: It must be too late here.. merging then.

**Comment by tridge** [comment]
looks good to me, applied, thanks!

**Comment by Susurrus** [comment]
Thanks for the quick turnaround time on this guys, appreciate it.
